packages/bundler/src/project.ts

```diff
@@ -261,6 +261,7 @@ export function projectFactory() {
       type NapiEndpoint = { __napiType: "Endpoint" };
 
       type NapiEntrypoints = {
+        apps?: NapiEndpoint[];
```

Comment:
we support a config apps to support bundler for application, can u please help to add a example under `examples/*` so we can debug the difference between with `libraries` and `apps`